packages/graphback-cli/src/templates/configTemplates.ts

```diff
@@ -1,48 +1,47 @@
 //tslint:disable: no-string-literal
 import { readFileSync, writeFileSync, existsSync } from 'fs';
-import { prompt as ask } from 'inquirer'
-import { safeDump } from 'js-yaml'
+import { prompt as ask } from 'inquirer';
+import { safeDump } from 'js-yaml';
 import { logInfo } from '../utils';
 
-const configFilesPath = `${__dirname}/resources/config`
+const configFilesPath = `${__dirname}/resources/config`;
 
-const dockerFilesPath = `${__dirname}/resources/docker`
+const dockerFilesPath = `${__dirname}/resources/docker`;
+
+const encoding = "utf8";
+const mongoDB = "MongoDB";
+const sqlite3 = "sqlite3";
+const postgreSQL = "PostgreSQL";
 /**
  * Different database choices
  */
 const getConfig = (database: string) => {
-  if (database === 'pg') {
-    return [readFileSync(`${configFilesPath}/postgres.json`, 'utf8'), readFileSync(`${dockerFilesPath}/postgres.yml`, 'utf8')]
-  } else if (database === 'MongoDB') {
-    return ["", readFileSync(`${dockerFilesPath}/mongodb.yml`, 'utf8')]
-  } else if (database === 'sqlite3') {
-    return [readFileSync(`${configFilesPath}/sqlite3.json`, 'utf8'), undefined]
+  if (database === postgreSQL) {
```

Comment:
this name was used for knex to identify database. If we use it here we will need to map it in the knex config

Reply:
Ah.. ignore me - we no longer actually put any db info into config! All good

Reply:
Yeap, I had to double check before doing it. 

Actually, this function could only return the docker-compose part as the first element of the array is never used.